fighthealthinsurance/websockets.py

```
@@ -73,3 +83,236 @@ async def receive(self, text_data):
+                    {
+                        "status": "generating",
+                        "message": "Starting to generate prior authorization proposals",
+                    }
+                )
+            )
+
+            # Update status
+            await self._update_prior_auth_status(prior_auth, "prior_auth_requested")
+
+            # Fetch the related created_for & creator fields
+            created_for = prior_auth.creator_professional_user
```

Ruff (0.8.2)

Local variable `created_for` is assigned to but never used

Remove assignment to unused variable `created_for`

(F841)